patroni/config.py

```diff
@@ -183,6 +197,13 @@ def _safe_copy_dynamic_configuration(self, dynamic_configuration):
                         config['postgresql'][name].update(self._process_postgresql_parameters(value))
                     elif name not in ('connect_address', 'listen', 'data_dir', 'pgpass', 'authentication'):
                         config['postgresql'][name] = deepcopy(value)
+            elif name == 'standby_cluster':
+                allowed_keys = self.__DEFAULT_CONFIG['standby_cluster'].keys()
+                expected = {
+                    k: v for k, v in (value or {}).items()
+                    if (k in allowed_keys and isinstance(v, six.string_types))
```

Comment:
Before trying it, I was looking at how this standby_cluster feature is implemented and just spotted that this one drops `create_replica_methods` which is a list, is it intentional?